backend/daemon/api/documents/v1alpha/documents.go

```diff
@@ -443,6 +458,82 @@ func (api *Server) DeletePublication(ctx context.Context, in *documents.DeletePu
 	return &emptypb.Empty{}, nil
 }
 
+// PushPublication implements the corresponding gRPC method.
+func (api *Server) PushPublication(ctx context.Context, in *documents.PushPublicationRequest) (*emptypb.Empty, error) {
+	if in.DocumentId == "" {
+		return nil, status.Errorf(codes.InvalidArgument, "must specify publication ID")
+	}
+
+	if in.Version == "" {
+		return nil, status.Errorf(codes.InvalidArgument, "must specify version")
+	}
+
```

Comment:
I'd suggest to check that the gateway URL is not empty.

Reply:
sure